openedx/core/djangoapps/labster/course/views.py

```diff
@@ -149,8 +150,12 @@ def course_handler(request, course_key_string=None):
 @authentication_classes((SessionAuthentication, TokenAuthentication))
 @permission_classes((IsAuthenticated,))
 def coach_list_handler(reuqest):
-    coach_list = ({
-        'full_name': ccx.coach.get_full_name(),
-        'email': ccx.coach.email,
-    } for ccx in CustomCourseForEdX.objects.all())
-    return Response(coach_list)
+    """
+    Returns a list of dicts with information about CCX coaches (full name, email).
+    """
+    coaches = User.objects.filter(courseaccessrole__role=CourseCcxCoachRole.ROLE)
```

Comment:
@auraz please take a look. 
Previous solution with `set` is replaced with this line.